src/widget/scroll.rs

```diff
@@ -147,11 +235,35 @@ impl<T: Data> Widget<T> for Scroll<T> {
         } else {
             None
         };
+
+        match event {
+            Event::AnimFrame(_) => {
+                // Animate scroll bars opacity
+                let opacity = self.scroll_bars.opacity.abs();
+                let diff = 0.004 / opacity;
+                self.scroll_bars.opacity -= diff;
```

Comment:
I think it'd be cleaner to use saturating subtraction here, to prevent opacity from going negative (though see you do have a `<=` check above, so the overall effect is the same).